transfers/amclientargs.py

```diff
@@ -127,6 +131,24 @@
         defaults.DEFAULT_TRANSFER_TYPE),
     default=defaults.DEFAULT_TRANSFER_TYPE,
     type=None)
+TRANSFER_SOURCE_OPT = Opt(
+    name='transfer-source',
+    metavar='TRANSFERSOURCE',
+    help='Transfer source UUID',
+    default=None,
+    type=None)
+SAVEAS_FILENAME = Opt(
+    name="saveas-filename",
+    metavar='SAVEASFILENAME',
+    help="A filename to save a file relative to a SIP to",
```

Comment:
Isn't the filename relative to the directory from where you're invoking the script?